services/cinema_hall.py

```diff
@@ -0,0 +1,18 @@
+from django.db.models import QuerySet
+
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet[CinemaHall]:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(
+        hall_name: str,
+        hall_rows: int,
+        hall_seats_in_row: int) -> CinemaHall:
+    return CinemaHall.objects.create(
+        name=hall_name,
+        rows=hall_rows,
+        seats_in_wor=hall_seats_in_row
```

Comment:
There is a typo in the `create_cinema_hall` function. The parameter `seats_in_wor` should be `seats_in_row` to match the model's field name.